arviz/plots/kdeplot.py

```diff
@@ -146,3 +168,72 @@ def fast_kde(x, cumulative=False, bw=4.5):
         density = cs_density / cs_density[-1]
 
     return density, xmin, xmax
+
+
+def fast_kde_2d(x, y, circular=False):
+    """
+    2D fft-based Gaussian kernel density estimate (KDE).
+
+    The code was adapted from https://github.com/mfouesneau/faststats
+
+    Parameters
+    ----------
+    x : Numpy array or list
+    y : Numpy array or list
+    circular: bool
+        If True, use circular boundaries. Defaults to False
+    Returns
+    -------
+    grid: A gridded 2D KDE of the input points (x, y)
+    xmin: minimum value of x
+    xmax: maximum value of x
+    ymin: minimum value of y
+    ymax: maximum value of y
+    """
+    x = np.asarray(x, dtype=float)
+    x = x[np.isfinite(x)]
+    y = np.asarray(y, dtype=float)
+    y = y[np.isfinite(y)]
+
+    xmin, xmax = x.min(), x.max()
+    ymin, ymax = y.min(), y.max()
+
+    len_x = len(x)
+    weights = np.ones(len_x)
+    n_x, n_y = [128, 128]  # [256, 256]
```

Comment:
What does the comment # [256, 256]  mean/do?

Reply:
This is the gridsize, I was experimenting with different values. I remove it. thanks!